### ISSUE TITLE
Estimating time remaining to complete

### ISSUE THREAD

**Issue by lgariv** [lgariv/Prog#17]
[NSProgress estimatedTimeRemaining](https://developer.apple.com/documentation/foundation/nsprogress/2868041-estimatedtimeremaining?language=objc)

**Comment by RuntimeOverflow** [comment]
@lgariv Estimated Time Remaining seems to be at 0 seconds at all times.

**Comment by RuntimeOverflow** [comment]
I'll push my code anyway with the estimated progress commented.

**Comment by RuntimeOverflow** [comment]
@lgariv Not really sure how you want to implement this when the estimated time remaining is nil. I think this is a clear indicator that iOS itself doesn't know how long it will take and it is almost impossible for us to find our own algorithm because while we may find the download speed and can calculate the remaining download time, it is almost impossible to find out how long the unpacking/installing will take unless there is another secret value indicating that.

**Comment by lgariv** [comment]
> @lgariv Not really sure how you want to implement this when the estimated time remaining is nil. I think this is a clear indicator that iOS itself doesn't know how long it will take and it is impossible for us to find out own algorithm because while we may find the download speed and can calculate the remaining download time, it is almost impossible to find out how long the unpacking/installing will take unless there is another secret value indicating that.

Agreed. For the time being, I'd like if you can revert / comment out (if you think it could be useful in the near future) your changes to show elapsed time, as timers are more resource heavy and it is not as useful as a remaining time indicator. They can also cause crashes if the function they call is dependent on a UIView (already happened to me a couple of times). Also if you can bring back the "Download paused" title to the notification when paused 👍🏼

**Comment by RuntimeOverflow** [comment]
I'll remove the code for the time elapsed completely, but I'll leave the label commented because maybe in the future we want to display additional text there, so we can just uncomment and use that label.
